doc/using-sourmash-a-guide.md

```diff
@@ -0,0 +1,147 @@
+and assembled genomes, transcriptomes, and metagenomes.  The high error
+rate of PacBio and Nanopore sequencing is problematic for k-mer based
+approaches and we have not yet explored how to tune parameters for
+this kind of sequencing.
+
+On a more practical note, `sourmash compute` should autodetect FASTA,
+FASTQ, whether they are uncompressed, gzipped, or bzip2-ed.  Nothing
+special needs to be done.
+
+## How should I prepare my data?
+
+Raw Illumina read data sets should be k-mer abundance trimmed to get rid of
+the bulk of erroneous kmers. We suggest a command like the following,
+using [trim-low-abund from the khmer project](https://peerj.com/preprints/890/) --
+
+```
+trim-low-abund.py -C 3 -Z 18 -V -M 2e9 <all of your input read files>
+```
+
+This is safe to use on genomes, metagenomes, and transcriptomes.  If you
+are working with large genomes or diverse metagenomes, you may need to
+increase the `-M` parameter to use more memory.
+
+See [the khmer docs for trim-low-abund.py](https://khmer.readthedocs.io/en/v2.1.2/user/scripts.html#trim-low-abund-py) and [the semi-streaming preprint](https://peerj.com/preprints/890/) for more information.
+
+For high coverage genomic data, you can do very stringent trimming with
+an absolute cutoff, e.g.
+
+```
+trim-low-abund.py -C 10 -M 2e9 <all of your input read files>
+```
+
+will eliminate all k-mers that appear fewer than 10 times in your data
+set.  This kind of trimming will dramatically reduce your sensitivity
+when working with metagenomes and transcriptomes, however, where there
+are always real low-abundance k-mers present.
+
+## Could you just give us the !#%#!$ command line?
+
+Sorry, yes! See below.
+
+### Computing signatures for read files:
+
+```
+trim-low-abund -C 3 -Z 18 -V -M 2e9 input-reads-1.fq input-reads-2.fq ...
+sourmash compute --scaled 1000 -k 21,31,51 input-reads*.fq.abundtrim \
+    --merge SOMENAME -o SOMENAME-reads.sig
+```
+
+The first command trims off low-abundance k-mers from high-coverage
+reads; the second takes all the trimmed read files, subsamples k-mers
+from them at 1000:1, and outputs a single merged signature named
+'SOMENAME' into the file `SOMENAME-reads.sig`.
+
+### Computing signatures for individual genome files:
+
+```
+sourmash compute --scaled 1000 -k 21,31,51 *.fna.gz --name-from-first
+```
+
+This command computes signatures for all `*.fna.gz` files, and names
+each signature based on the first FASTA header in each file (that's
+what the option `--name-from-first` does). The signatures will be placed
+in `*.fna.gz.sig`.
+
+### Computing signatures from a collection of genomes in a single file:
+
+```
+sourmash compute --scaled 1000 -k 21,31,51 file.fa --singleton
+```
+
+This computes signatures for all individual FASTA sequences in `file.fa`,
+names them based on their FASTA headers, and places them all in a single
+`.sig` file, `file.fa.sig`.  (This behavior is triggered by the option
```

Comment:
`file.fa.sig` should be `FASTAHEADER.sig`?

Reply:
no, the output filename is file.fa.sig based on file.fa.